index.html

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-Mobile</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <div class="logo">
+          <a href="#">Welcome</a>
+        </div>
+
+        <div class="vector">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+      </nav>
+    </header>
+
+    <main>
```

Comment:
Greate Ui! background color should be `#fff`. please consider updating the background color as per the design.👍 
| Your | Design|
| --- | --- |
| ![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/55840999/ce8e2eae-2d0b-42d1-8f51-f0897752c4c4) | ![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/55840999/f06c0800-c469-4cc6-81e6-9fe95caf6fd1)   |